functions/httpserver.go

```diff
@@ -22,19 +23,26 @@ type Network struct {
 	Server config.Server
 }
 
+const DEFAULT_HTTP_SERVER_PORT = "18095"
+const DEFAULT_HTTP_SERVER_ADDR = "127.0.0.1"
```

Comment:
The convention in Go is to use MixedCaps or mixedCaps (simply camelCase) rather than underscores to write multi-word names

Reply:
fixed